articles/anomaly-detection/concepts/credential-stuffing.md

```diff
@@ -0,0 +1,43 @@
+---
+description: Understand how Auth0 detects anomalies to stop malicious attempts to access your application, alert you and your users of suspicious activity, and block further login attempts. 
+topics:
+    - security
+    - anomaly-detection
+    - credential-stuffing
+contentType: concept
+useCase: anomaly-detection
+---
+# Automated Attack and Credential Stuffing Protection
```

Comment:
Is this really "Automated Attack" *and* "Credential Stuffing" protections?  Again, use of Automated Attack is unclear - is it the same?  Is Credstuff a type of automated attack?  If so, why "and" ?  If not, why are we including Automated Attack here?